app/jsx/components/FilterComp.jsx

```diff
@@ -54,7 +54,7 @@ class FilterComp extends React.Component {
             ) }
           </ul>
         </details>
-        {/* <a href="" className="c-filter__tips">Search tips</a> */}
+        <a href="https://help.escholarship.org/support/solutions/articles/9000148939-using-advanced-search-beta-" className="c-filter__tips">Search tips</a>
```

Comment:
Nicely written and clear. Question: it says "The + and - operators are not officially supported by eScholarship" ... does that mean they're supported but unofficial? Shouldn't we just not support them at all?

Reply:
We don't support them at all. I'll ask Justin to remove the word official